hw8/code/classes.py

```diff
@@ -0,0 +1,185 @@
+from Bio.Seq import Seq
+from Bio import SeqIO
+from Bio.SeqUtils import GC
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+
+class Employee:
+
+    def __init__(self, name, age, work_experience):
+        self.name = name
+        self.age = age
+        self.work_experience = work_experience
+
+    def __str__(self):
+        return 'Information about employee: {}'.format(self.name)
+
+    def proposed_position(self):
+        '''
+        Shows the position the candidate is applying for
+        :return: str, position name
+        '''
+        if self.work_experience <= 1:
+            position = 'Junior DS'
+        elif 1 < self.work_experience <= 3:
+            position = 'Middle DS'
+        else:
+            position = 'Senior DS'
+        return position
+
+    def proposed_salary(self, position):
+        '''
+        Shows salary the candidate wants to have
+        :param position: str, name of the position
+        :return: int, salary in dollars
+        '''
+        if position == 'Junior DS':
+            salary = 1000
+        elif position == 'Middle DS':
+            salary = 5000
+        else:
+            salary = 10000
+        return salary
+
+    def view_info(self):
+        '''
+        Shows basic information about candidate
+        :return: str with information
+        '''
+        position = self.proposed_position()
+        salary = self.proposed_salary(position)
+        return f"Name: {self.name}\nAge: {self.age} years\nPosition: {position}\nSalary {salary} $"
+
+
+class SetModified(set):
+
+    def __init__(self, *args):
+        self.positive_numbers = [arg for arg in args if isinstance(arg, (int, float)) and arg > 0]
+        super().__init__(self.positive_numbers)
```

Comment:
Записывать это в атрибут не нужно
```suggestion
        positive_numbers = [arg for arg in args if isinstance(arg, (int, float)) and arg > 0]
        super().__init__(positive_numbers)
```